src/Components/CreateImageEntry.jsx

```diff
@@ -30,21 +32,55 @@ class CreateImageEntry extends Component {
     })
   }
 
+  geolocationDataAddress = () => {
+    Geocode.setApiKey(process.env.REACT_APP_API_KEY_GOOGLE_MAPS)
+    Geocode.fromAddress(this.state.userInputAddress).then(
+      response => {
+        const { lat, lng } = response.results[0].geometry.location;
+        this.setState({
+          latitude: lat,
+          longitude: lng
+
+        })
+        this.geolocationDataCoords()
+      },
+      error => {
+        console.error(error);
+      }
+    )
+  }
+
+  geolocationDataCoords = () => {
+    Geocode.setApiKey(process.env.REACT_APP_API_KEY_GOOGLE_MAPS)
```

Comment:
Do we need this code repetition? Can this value be set in the constructor?

Reply:
will be done in another chore PR